### DISCUSSION TITLE
Not Really an Issue per se just a burning Question regarding the Simple Alpha Zero implementation for Othello

### DISCUSSION THREAD

**Discussion by AryanSarswat** [suragnair/alpha-zero-general#236] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/65006917/108508840-644a1f80-72f7-11eb-8f19-0aff31e66811.png)

In this particular function where you are returning the symmetrical board states, the line where you are updating the list l, you 
add (newB, list(newPi.ravel()) + [pi[-1]]).

Why do you add the old policy vector to the new rotated/mirrored policy vector?

**Comment by suragnair** [comment, marked as answer]
By doing all 4 rotations it keeps the original orientation as well. Does that answer your question?

**Reply by AryanSarswat** [reply]
Ahhh right, that makes sense, Thank you!
